bin/plotting/pycbc_ifar_catalog

```diff
@@ -236,9 +241,10 @@ if opts.open_box:
 if opts.open_box:
     # scale the plot around the foreground
     pylab.ylim(0.7, 0.05 * len(fore_cumnum))
-    pylab.xlim(30 * min(fore_ifar), 3 * max_ifar)
+    pylab.xlim(min(expected_ifar) / 30, 3 * max_ifar)
```

Comment:
why is this an improvement?

Reply:
This is a leftover from before I did the switch to counting louder events below threshold instead of extrapolating the fit backwards.

Basically the minimum fore_ifar could have been _really_ low, and so anything below the minimum expected_ifar was essentially rubbish.

However stuff at really low IFAR is now telling us that something has gone wrong, so I will revert it